src/PayID/pay-id-client.ts

```diff
@@ -0,0 +1,28 @@
+import PayIDError from './pay-id-error'
+
+/* eslint-disable @typescript-eslint/require-await */
+// TODO(keefertaylor): Enable lint rules when method is implemented.
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable max-classes-per-file */
+
+/**
+ * A client for PayID.
+ *
+ * @warning This class is experimental and should not be used in production applications.
+ * TODO(keefertaylor): Export this class in index.ts when it's ready for external consumption.
+ */
+export default class PayIDClient {
+  /**
+   * Retrieve the XRP Address authorized with a PayID.
+   *
+   * @note The returned value will always be in an X-Address format.
```

Comment:
Not X-address + (classic address + dest. tag)?

Reply:
Xpring SDK only serves X-addresses. I'm happy to revisit this API with you at another point but it's all the same information. 

Serving redundant information feels confusing to users.